components/dropdown/dropdown-menu.js

```diff
@@ -167,7 +167,7 @@ if (usePopoverMixin) {
 		}
 
 		#handleSelect(e) {
-			if (['D2L-MENU-ITEM', 'D2L-MENU-ITEM-LINK', 'D2L-SELECTION-ACTION-MENU-ITEM'].indexOf(e.target.tagName) < 0) {
+			if (['D2L-MENU-ITEM', 'D2L-MENU-ITEM-LINK', 'D2L-SELECTION-ACTION-MENU-ITEM', 'D2L-BUTTON-SPLIT-ITEM'].indexOf(e.target.tagName) < 0) {
```

Comment:
Doesn't have to be now, but it might be nice to just check if it implements `MenuItemMixin` instead of doing this... @GZolla did something similar recently [with ButtonMixin](https://github.com/BrightspaceUI/core/pull/5509).

Reply:
Yeah, that might be possible. I'm curious why we haven't done that yet, since we've done it in a few places. 🤔